app/javascript/react/components/Button/Button.style.tsx

```diff
@@ -0,0 +1,25 @@
+import styled, { css } from "styled-components";
+import { buttonText } from "../../assets/styles/Typography";
+import { lightGray } from "../../assets/styles/colors";
+
+const buttonGeneralStyles = css`
+  ${buttonText}
+  height: 42px;
+  border-radius: 5px;
+  padding: 16px;
+  align-items: center;
+  gap: 10px;
+`;
+
+const Button = styled.button`
```

Comment:
What is the point of this? Is this some kind of a middle thing that does nothing more? If so maybe we can shorten this and change HeaderButton

```
const HeaderButton = styled.button`
  ${buttonGeneralStyles}
  justify-content: flex-end;
  display: flex;
  border: 1px solid ${lightGray};
  background-color: transparent;
`;
```

Reply:
Hmmm, tbh I had some errors and I was trying different things. Also I had some code inside the `Button` and then I moved it but left the `Button` to avoid the previous errors.  But now I see it's working fine when I remove this `Button` 🤡 so I guess I'll just remove it